java/integTest/src/test/java/glide/TransactionTestUtilities.java

```diff
@@ -279,12 +279,23 @@ private static Object[] listCommands(BaseTransaction<?> transaction) {
         if (REDIS_VERSION.isGreaterThanOrEqualTo("7.0.0")) {
             transaction
                     .lpush(listKey4, new String[] {value1, value2, value3, value1, value2, value3})
-                    .lmpop(new String[] {listKey4}, PopDirection.LEFT)
-                    .lmpop(new String[] {listKey4}, PopDirection.LEFT, 2L)
-                    .blmpop(new String[] {listKey4}, PopDirection.LEFT, 0.1)
-                    .blmpop(new String[] {listKey4}, PopDirection.LEFT, 2L, 0.1);
+                    .lmpop(new String[] {listKey4}, ListDirection.LEFT)
+                    .lmpop(new String[] {listKey4}, ListDirection.LEFT, 2L)
+                    .blmpop(new String[] {listKey4}, ListDirection.LEFT, 0.1)
+                    .blmpop(new String[] {listKey4}, ListDirection.LEFT, 2L, 0.1);
         } // listKey4 is now empty
 
+        if (REDIS_VERSION.isGreaterThanOrEqualTo("6.2.0")) {
+            transaction
+                    .del(new String[] {listKey5})
```

Comment:
Delete list 4 too - it could be non empty before this block.
Or start 2 new keys

Reply:
decided to start new keys, since `del` would return differently depending on if `listKey4` was used or not and makes expected value tricky